eagle-core/eagle-policy/eagle-policy-base/src/main/java/org/apache/eagle/policy/dao/PolicyDefinitionDAO.java

```diff
@@ -21,6 +21,7 @@
 import java.util.Map;
 
 import org.apache.eagle.alert.entity.AbstractPolicyDefinitionEntity;
+import org.apache.eagle.alert.entity.AlertDefinitionAPIEntity;
```

Comment:
Why need to have AlertDefinitionAPIEntity in PolicyDefinitionDAO? Ideally PolicyDefinitionDAO suppose not knowing any the specific subtype of AbstractPolicyDefinitionEntity.